000_introduction/000_template.tex

```diff
@@ -13,7 +13,7 @@ \section{Sections, Subsections, and Labels}\label{sections_subsections_labels}
 
 You should always label your sections by using \textbackslash label\{your label\}. If you have labels, you can refer to the corresponding section using \textbackslash Cref\{your\_label\}.
 
-
+Implementing a glossary word can be done using \textbackslash gls\{glossary\_word\}. Other Examples (i.e. capitalisation) can be discovered at \url{https://www.overleaf.com/learn/latex/Glossaries}
```

Comment:
Should we allow teams to add glossary words? Might want to separate team-specific terminology perhaps - or have their own terminologies/glossary at the beginning of their section

Reply:
I think it's better if every team is consistent with the terminology... any redefinitions or term changes will just confuse readers.